driver/src/main/java/org/kaazing/k3po/driver/internal/Robot.java

```diff
@@ -93,6 +96,8 @@
 
     private final ChannelHandler closeOnExceptionHandler = new CloseOnExceptionHandler();
 
+    private Map<String, Barrier> barriersByName;
```

Comment:
ok, made it a ConcurrentMap that got passed into constructor of GenerateConfigurationVisitor.  Like it better then it was